packages/components/package.json

```diff
@@ -26,7 +26,7 @@
 	"module": "build-module/index.js",
 	"exports": {
 		".": {
-			"types": "./build-types/index.d.ts",
+			"types": "./build-types",
```

Comment:
Do you think this change should be part of this PR?

Reply:
@youknowriad Can you elaborate what's mandatory about this? I ran into the opposite issue in #72978 with the changes here causing resolution to fail.